GuruLoader/project.json

```diff
@@ -3,12 +3,11 @@
 
   "dependencies": {
     "HtmlAgilityPack.NetCore": "1.5.0.1",
-    "NETStandard.Library": "1.6.0"
+    "NETStandard.Library": "1.5.0-rc2-24027"
```

Comment:
Confusingly, this can actually stay 1.6; only the moniker in the frameworks element needs to change to 1.5. Doing that will use the .NET Standard 1.5 subset of the .NET Standard library (whose current version unfortunately happens to be 1.6).